modules/test/playwright/pages/site-admin-web/SiteSettingsLocalizationPage.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+import {SiteSettingsPage} from './SiteSettingsPage';
+
+export class SiteSettingsLocalizationPage {
+	readonly page: Page;
+	readonly saveButton: Locator;
+	readonly siteSettingsPage: SiteSettingsPage;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.saveButton = page.getByRole('button', {name: 'Save'});
+		this.siteSettingsPage = new SiteSettingsPage(page);
+	}
+
+	async goto() {
+		await this.siteSettingsPage.goToSiteSetting('Localization');
+	}
+
+	async saveConfiguration() {
+		this.saveButton.click();
+		await waitForSuccessAlert(this.page);
+	}
+
+	async selectCustomDefaultLanguageOption() {
+		await this.page
+			.getByLabel(
```

Comment:
No sure why you did not add this locator to the constructor.

Reply:
this was some confusion in my part, first this was a generic function that received a label and returned the correspondent element, but after i decided to separate them since it's only two different radio buttons. i should make them locators in the constructor indeed.